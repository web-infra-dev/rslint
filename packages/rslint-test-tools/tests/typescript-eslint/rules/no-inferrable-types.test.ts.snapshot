exports[`no-inferrable-types > invalid 1`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-inferrable-types",
      "messageId": "noInferrableType",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 7
        },
        "end": {
          "line": 1,
          "column": 8
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`no-inferrable-types > invalid 2`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-inferrable-types",
      "messageId": "noInferrableType",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 7
        },
        "end": {
          "line": 1,
          "column": 8
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`no-inferrable-types > invalid 3`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-inferrable-types",
      "messageId": "noInferrableType",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 7
        },
        "end": {
          "line": 1,
          "column": 8
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`no-inferrable-types > invalid 4`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-inferrable-types",
      "messageId": "noInferrableType",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 7
        },
        "end": {
          "line": 1,
          "column": 8
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1
}
`;

exports[`no-inferrable-types > invalid 5`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 1
}
`;
