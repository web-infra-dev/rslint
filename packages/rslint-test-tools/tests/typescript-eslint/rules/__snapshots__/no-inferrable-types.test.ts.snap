// Rstest Snapshot v1

exports[`no-inferrable-types > invalid 1`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "messageId": "noInferrableType",
      "range": {
        "end": {
          "column": 8,
          "line": 1,
        },
        "start": {
          "column": 7,
          "line": 1,
        },
      },
      "ruleName": "no-inferrable-types",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-inferrable-types > invalid 2`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "messageId": "noInferrableType",
      "range": {
        "end": {
          "column": 8,
          "line": 1,
        },
        "start": {
          "column": 7,
          "line": 1,
        },
      },
      "ruleName": "no-inferrable-types",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-inferrable-types > invalid 3`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "messageId": "noInferrableType",
      "range": {
        "end": {
          "column": 8,
          "line": 1,
        },
        "start": {
          "column": 7,
          "line": 1,
        },
      },
      "ruleName": "no-inferrable-types",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-inferrable-types > invalid 4`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "Type bigint trivially inferred from a bigint literal, remove type annotation.",
      "messageId": "noInferrableType",
      "range": {
        "end": {
          "column": 8,
          "line": 1,
        },
        "start": {
          "column": 7,
          "line": 1,
        },
      },
      "ruleName": "no-inferrable-types",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-inferrable-types > invalid 5`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 1,
  "ruleCount": 1,
}
`;
